config/locales/en.yml

```diff
@@ -1609,6 +1609,14 @@ en:
           Hello - Thank you for using FileYourStateTaxes! Any issues you may have experienced while using our tool have now been resolved. To continue filing your state taxes, log in here: %{login_link}
 
           Questions? Reply to this text.
+      post_deadline_reminder:
+        email:
+          body: "Hi %{primary_first_name},\n\nYou haven't submitted your state tax return yet. As a reminder, state taxes are due by April 15. Make sure to submit your state  tax return by April 15 in order to avoid late filing fees from %{state_name}. \n\nGo to fileyourstatetaxes.org/en/us/login-options to finish them now.\n\nNeed help? Reply to this email.\n\nBest,\nThe FileYourStateTaxes team\n"
+          subject: Don't forget to finish your state tax return by April 15!
```

Comment:
I think the subject should be `Make sure to finish filing your state taxes as soon as possible.` according to https://docs.google.com/spreadsheets/d/1pn0ib-gAYgZQJn-KmQJgch5WreS7qvNWJApLHrfQwxg/edit#gid=0